src/components/RegisterLoginModal/RegisterLoginModal.jsx

```diff
@@ -43,9 +44,20 @@ function RegisterLoginModal() {
             bgcolor: 'background.paper',
           }}
         >
-          <RegisterLoginHeader checked={checked} onChange={handleChange} />
+          <IconButton aria-label="Close" className={styles.closeButton} onClick={handleClose}>
+            <CloseIcon />
+          </IconButton>
+          <RegisterLoginHeader checked={checked} onChange={handleChange} onClick={handleClose} />
           {checked ? <RegisterForm fn={changeState} /> : <LoginForm fn={changeState} />}
-          {state ? <ActionAlert fn={setState} children={state} /> : null}
+          {state === 'Success' ? (
+            <ActionAlert severity="success" fn={setState}>
+              {state}
```

Comment:
The text of an error doesn't show up - `children` are not rendered inside `ActionAlert`. The error could also be cleared if one switches between the login and registers forms.